.github/workflows/roman_ci.yml

```
@@ -69,18 +69,18 @@ jobs:
         DD_GIT_COMMIT_SHA: ${{ github.sha }}
         DD_GIT_BRANCH: ${{ github.ref_name }}
       cache-path: /tmp/data/
-      cache-key: data-${{ needs.webbpsf_data_cache.outputs.cache_key }}-${{ needs.crds_context.outputs.context }}
-      cache-restore-keys: ${{ needs.webbpsf_data_cache.outputs.cache_key }}
+      cache-key: data-${{ needs.stpsf_data_cache.outputs.cache_key }}-${{ needs.crds_context.outputs.context }}
+      cache-restore-keys: ${{ needs.stpsf_data_cache.outputs.cache_key }}
       envs: |
-        - linux: py311-oldestdeps-webbpsf-cov
+        - linux: py311-oldestdeps-stpsf-cov
           pytest-results-summary: true
-        - linux: py311-webbpsf-nolegacypath
+        - linux: py311-stpsf-nolegacypath
           pytest-results-summary: true
-        - linux: py312-ddtrace-webbpsf
+        - linux: py312-ddtrace-stpsf
           pytest-results-summary: true
-        - macos: py312-ddtrace-webbpsf
+        - macos: py312-ddtrace-stpsf
           pytest-results-summary: true
-        - linux: py312-webbpsf-cov
+        - linux: py312-stpsf-cov
           coverage: codecov
           pytest-results-summary: true
-        # - linux: py313-webbpsf  # Distabled until we support Python 3.13
+        # - linux: py313-stpsf  # Distabled until we support Python 3.13
```

```suggestion
        - linux: py313-stpsf
```
Might as well enable it since python 3.13 is supported